src/main/java/de/baleipzig/iris/ui/view/recognition/RecognitionView.java

```diff
@@ -50,6 +52,9 @@
 
     private final HorizontalLayout recognitionMainLayout = new HorizontalLayout();
 
+    private final DrawingLayout drawingLayout= new DrawingLayout();
+    private final DigitImageLayout digitImageLayout = new DigitImageLayout();
+
     private boolean infoPanelVisible = false;
 
     @Override
```

Comment:
die initRecognitionLayout-Methode ist viel zu lang. hier wuerde ich ein runterbrechen in kleinere methoden empfehlen. in anbetracht der fortgeschrittenen zeit, stelle ich dir aber in meiner unendlichen güte (XD) frei, das zu ignorieren.